docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -79,38 +81,32 @@ In JupyterHub, you should see the new kernel, `my_env`, available.
 
 Try running your JupyterHub notebook using the `my_env` kernel you just installed.  You can change the kernel of an existing notebook by going to Kernel->Change Kernel...
```

Comment:
the arrow -> looks kind of bad. I don't know if there is a way to make it bigger or add spaces around it maybe?

Reply:
done